src/main/java/gift/login/controller/LoginController.java

```diff
@@ -0,0 +1,31 @@
+package gift.login.controller;
+
+import gift.login.service.LoginService;
+import jakarta.validation.constraints.NotNull;
+import java.net.URI;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.ui.Model;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestClient;
+
+@RestController
+public class LoginController {
+    private final LoginService loginService;
```

Comment:
개발자는 단순히 코드 작성하는 역할이 아니라, 협업을 통해 비즈니스를 만드는 역할입니다ㅎㅎ
개행은 사소해보이지만 함께 일하는 다른 개발자와의 협업에서 중요한 포인트이므로, 잘 신경쓰는 것이 좋은 것 같다고 생각합니다ㅎㅎ
```suggestion

    private final LoginService loginService;
```